libcodechecker/libhandlers/parse.py

```diff
@@ -204,8 +225,36 @@ def main(args):
                     os.chdir(metadata_dict['working_directory'])
 
             _, _, files = next(os.walk(input_path), ([], [], []))
+
             for f in files:
-                parse(os.path.join(input_path, f), context, metadata_dict,
-                      suppress_handler, 'print_steps' in args)
+                report_stats = parse(os.path.join(input_path, f),
+                                     context,
+                                     metadata_dict,
+                                     suppress_handler,
+                                     'print_steps' in args)
+
+                severity_stats.update(Counter(report_stats.get('severity',
+                                              {})))
+                file_stats.update(Counter(report_stats.get('files', {})))
+
+        if file_stats or severity_stats:
+            print("----==== Summary ====----")
+            vals = [[os.path.basename(k), str(v)] for k, v in
```

Comment:
You are uniqueing the result by file path but you are writing out only the filename in the summary table.

E.g.: Suppose we have 2 files with the same name (a/myheader.h, b/myheader.h) and both of them contain several reports. The result will looks like this:
```bash
----==== Summary ====----
-------------------------
Filename   | Report count
-------------------------
myheader.h | 1           
myheader.h | 1           
-------------------------
-----------------------
Severity | Report count
-----------------------
HIGH     | 2           
-----------------------
----=================----
```
Wouldn't it be better to write out file path instead of file name in the result table?

Reply:
I thought about that but a file path can be very long which might break the table layout.